tests/e2e/tracks-page.test.ts

```diff
@@ -103,10 +81,10 @@ test.describe('Tracks Page', () => {
       await searchInput.fill(searchTerm);
 
       await page.waitForTimeout(500);
-      
+
       const noTracksMessage = page.getByText('No tracks found');
       const isNoTracksVisible = await noTracksMessage.isVisible();
-      
+
       if (isNoTracksVisible) {
```

Comment:
I think it would probably be better to not add these if statements, but rather fail the test entirely, if the element is not found:

(same thing for other tests)

```suggestion
      await expect(noTrackMessage).toBeVisible()
```

Reply:
Fixed